llama.cpp

```diff
@@ -2584,11 +2651,9 @@ static struct ggml_cgraph * llm_build_llama(
     struct ggml_init_params params = {
         /*.mem_size   =*/ buf_compute.size,
         /*.mem_buffer =*/ buf_compute.data,
-        /*.no_alloc   =*/ false,
+        /*.no_alloc   =*/ true,
     };
 
-    params.no_alloc = true;
-
```

Comment:
This is an OK change, but either do it for all `llm_build_` functions or keep the original and we'll refactor this later. Want to have the same pattern used everywhere

Reply:
Gotcha, made this change throughout.